lightning/src/ln/channelmanager.rs

```diff
@@ -918,7 +918,7 @@ impl<ChanSigner: ChannelKeys, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref>
 
 	/// Force closes a channel, immediately broadcasting the latest local commitment transaction to
 	/// the chain and rejecting new HTLCs on the given channel.
-	pub fn force_close_channel(&self, channel_id: &[u8; 32]) {
+	pub fn force_close_channel(&self, channel_id: &[u8; 32]) -> Result<(), APIError>{
```

Comment:
It would be nice to update the docs here to state that we only fail in case the channel is not found.

Reply:
Rebased in 0e83723